src/py/flwr/cli/new/templates/app/pyproject.baseline.toml.tpl

```diff
@@ -0,0 +1,142 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "$package_name"
+version = "1.0.0"
+description = ""
```

Comment:
Do we expect users to write something here? Should we add a TODO?

Reply:
yes, just mentioned it in the `README.md` that gets generated.